### PR TITLE
fix(json): adds serializer for translations

### PR DESCRIPTION
**Pull request by scefali**
This PR enables serialization of certain Django objects like lazy translations. Docs for this here: https://docs.djangoproject.com/en/1.8/topics/serialization/

### CODE REVIEW

**Review by evanpurkhiser — COMMENTED** [review]
Looks fine, but curious what more django knowledgeable people think.

**Review by wedamija — APPROVED** [review]
LGTM. I'm not sure if there are cases where a Django `Promise` can get to the serializer when it doesn't contain a translation, but we can deal with that if it happens.

File: tests/sentry/utils/json/tests.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import uuid
 
 from enum import Enum
+from django.utils.translation import ugettext_lazy as _
```
Comment (wedamija) [inline tests/sentry/utils/json/tests.py:9]: Nit: Sort

File: src/sentry/utils/json.py [hunk 2]
```diff
@@ -11,6 +11,8 @@
 from bitfield.types import BitHandler
 from django.utils.timezone import is_aware
 from django.utils.html import mark_safe
+from django.utils.functional import Promise
+from django.utils.encoding import force_text
```
Comment (wedamija) [inline src/sentry/utils/json.py:15]: Nit: Sort

**Pull request merged**